CarRental/CarRental.Web/Views/Search/_AvailableOptionsForYearFrom.cshtml

```diff
@@ -13,13 +13,13 @@
     string modelAlert = @Model == null ? "Model=null" : "Model != null";
 
 }
-<option value="">-Select production year-</option>
+<option value="">-Production year-</option>
```

Comment:
Jak się pomniejszy okno to nie widać
może lepiej krócej: -Year- , albo odpowiednio -Year From- i -Year To-
bo wtedy nawet przy zmniejszonym oknie widać początkowe litery